las/client.py

```diff
@@ -813,6 +933,95 @@ def delete_model(self, model_id: str) -> Dict:
         """
         return self._make_request(requests.delete, f'/models/{model_id}')
 
+    def create_data_bundle(self, model_id, dataset_ids, **optional_args) -> Dict:
+        """Creates a data bundle, calls the POST /models/{modelId}/dataBundles endpoint.
+
+        :param model_id: Id of the model
+        :type model_id: str
+        :param dataset_ids: Dataset Ids that will be included in the data bundle
+        :type dataset_ids: List[str]
+        :param name: Name of the data bundle
+        :type name: Optional[str]
+        :param description: Description of the data bundle
+        :type description: Optional[str]
+        :return: Data Bundle response from REST API
+        :rtype: dict
+
+        :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
+ :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
+        """
+
+        body = {'datasetIds': dataset_ids}
+        body.update(**optional_args)
+        return self._make_request(requests.post, f'/models/{model_id}/dataBundles', body=body)
+
+    def list_data_bundles(
+        self,
+        model_id,
+        *,
+        max_results: Optional[int] = None,
+        next_token: Optional[str] = None,
+    ) -> Dict:
+        """List data bundles available, calls the GET /models/{modelId}/dataBundles endpoint.
+
+        :param model_id: Id of the model
+        :type model_id: str
+        :param max_results: Maximum number of results to be returned
+        :type max_results: Optional[int]
+        :param next_token: A unique token for each page, use the returned token to retrieve the next page.
+        :type next_token: Optional[str]
+        :return: Data Bundles response from REST API
+        :rtype: dict
+
+        :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
+ :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
+        """
+        params = {
+            'maxResults': max_results,
+            'nextToken': next_token,
+        }
+        return self._make_request(requests.get, f'/models/{model_id}/dataBundles', params=params)
+
+    def update_data_bundle(
+        self,
+        model_id: str,
+        data_bundle_id: str,
+        **optional_args,
```

Comment:
There is some inconsistencies in when * is used. Should we refactor this across the SDK and publish a new major version?

Reply:
will fix in other PR